### PR TITLE
Add help button in header for logged in users

### PR DESCRIPTION
**Pull request by fjvallarino**
I understood we're leaving the Priority Support removal for a future card, so I'm requesting review as is.

@Rise-Vision/apps please review

### CODE REVIEW

File: src/js/controllers/ctr-zendesk-button.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+angular.module("risevision.common.header")
+
+.controller("ZendeskButtonCtrl", ["$scope", "userState", "zendesk",
```
Comment (alex-deaconu) [inline src/js/controllers/ctr-zendesk-button.js]: @fjvallarino I don't think userState is used here. Thanks.

File: src/scss/core/global.scss [hunk 2]
```diff
@@ -305,9 +305,9 @@ ul.social {
 .btn-default svg {
   margin-left: 5px;
 }
-.btn svg path {
-  fill: #666;
-}
+// .btn svg path {
+//   fill: #666;
+// }
```
Comment (alex-deaconu) [inline src/scss/core/global.scss]: @fjvallarino @RV-Adrian should we remove this code rather than comment it out? Thanks.

**Comment by alex-deaconu** [comment]
@fjvallarino 2 minor cleanup comments. Otherwise PR looks good. Thanks for the fixes 👍

File: src/scss/core/global.scss (hunk 2 above)
Comment (RV-Adrian) [reply src/scss/core/global.scss]: @fjvallarino @alex-deaconu 
Yes, let's remove it so we can deploy. I couldn't find a purpose for it (possibly for an old button style)

**Comment by alex-deaconu** [comment]
@fjvallarino looks good 👍

**Pull request merged**